cdap-ui/app/directives/flow-graph/flow.js

```diff
@@ -184,7 +199,14 @@ module.directive('myFlowGraph', function (d3, dagreD3, $state, $filter) {
          * Handles node click and sends to flowlet page.
          */
         function handleNodeClick(nodeId) {
-          $state.go('flows.detail.runs.detail.flowlets.detail', {flowletId: nodeId});
+          handleHideTip(nodeId);
+          var instance = instanceMap[nodeId];
+          if (instance.type === 'STREAM') {
+            $state.go('flows.detail.runs.tabs.status.streamsDetail', {streamId: nodeId});
+          } else {
+            $state.go('flows.detail.runs.tabs.status.flowletsDetail', {flowletId: nodeId});
```

Comment:
This won't draw a workflow diagram, i thought those were different. How could I change this to work for workflows?